poster_data/choice.md

```
@@ -2114,3 +2114,168 @@
+--- 北海道/登別市
+1. noboribetsu_poster_full_normalized.csv (8.8 KB)
+2. 登別市_normalized.csv (8.7 KB)
+-> choose 登別市_normalized.csv (shortest filename)
+
+--- 北海道/置戸町
+1. old置戸町_normalized.csv (1.9 KB)
+2. 置戸町_normalized.csv (1.9 KB)
+-> choose 置戸町_normalized.csv (shortest filename)
+
+📎 Append files to be loaded:
+  + 苫小牧市_normalized_20250714_append.csv (3.4 KB)
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)